docsets/XMake/docset.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "XMake",
-    "version": "2.6.2",
+    "version": "2.6.5",
     "archive": "XMake.tgz",
```

Comment:
The archive is named `XMake.tgz` here, but the file you added is named `Xmake.tgz`.